README.md

```diff
@@ -68,7 +68,7 @@ docker run -e LINK_UP_USERNAME="mail@example.com" \
            -e LINK_UP_TIME_INTERVAL="5" \
            -e LINK_UP_REGION="EU" \
            -e NIGHTSCOUT_URL="nightscout.yourdomain.com" \
-           -e NIGHTSCOUT_API_TOKEN="librelinku-123456789abcde" \
+           -e NIGHTSCOUT_API_TOKEN=$(echo -n "foo-bar-baz" | sha1sum | cut -d ' ' -f 1) \
```

Comment:
Instead of "foo-bar-baz" we should use the same example token value ("librelink...") as in all the other parts to keep the documentation consistent.